src/components/Project/Releases/index.js

```diff
@@ -43,10 +43,10 @@ class ReleaseView extends React.Component {
     })
 
     const orderedExtensions = filteredExtensions.sort(function(a, b){
-        if(b.extension.type === "workflow"){
-            return 1
+        if(a.extension.created < b.extension.created){
```

Comment:
why do we sort based on extension create time? This means they will show up in the order installed on the project, correct?